tests/readBison.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+readBison() test
```

Comment:
Could we have yet a test where an exception is thrown? Like perhaps feed some random garbage, etc. or something that would certainly fail it?

Thanks